pages/myAccount.page.ts

```diff
@@ -0,0 +1,11 @@
+import { Locator, Page } from "@playwright/test";
+
+export class MyAccountPage {
+    readonly pageHeader: Locator;
+    readonly pageInfoAccount: Locator;
+
+    constructor(page: Page) {
+        this.pageHeader = page.locator("h1.page-heading");
+        this.pageInfoAccount = page.locator("p.info-account");
+    }
+}
```

Comment:
please change all page object to functions page object

Reply:
Done.